src/main/java/org/opentripplanner/index/IndexGraphQLSchema.java

```diff
@@ -139,6 +141,25 @@
         .value("WALK", TraverseMode.WALK, "WALK")
         .build();
 
+    public static GraphQLEnumType filterModeEnum = GraphQLEnumType.newEnum()
```

Comment:
It should be available. See `org.opentripplanner.routing.car_park.CarPark`, `org.opentripplanner.routing.vertextype.ParkAndRideVertex` and `org.opentripplanner.routing.edgetype.ParkAndRideLinkEdge`